Gemfile

```diff
@@ -50,7 +50,7 @@ gem 'bootsnap', require: false
 
 # Use Sass to process CSS
 # gem "sassc-rails"
-
+# gem 'bullet', group: 'development'
```

Comment:
- [ ] While you did a fantastic job solving the `N+1` problems, it is crucial to [NOT use the bullet gem and to add screenshots of the logs before and after the fix to the PR description](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Do%20not%20use,after%20the%20fix). :+1:

Reply:
@IngDavidTM  thanks, i have fixed required changes